code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityExtension.java

```diff
@@ -318,7 +318,7 @@ void handleRemoveIdentity(@NonNull final Event event) {
 	 * @param event the identity request {@link Event}
 	 */
 	private void handleGetIdentifiersRequest(@NonNull final Event event) {
-		final Map<String, Object> xdmData = state.getIdentityProperties().toXDMData(false);
+		final Map<String, Object> xdmData = state.getIdentityProperties().toXDMData(true);
```

Comment:
This is the only use of `toXDMData(true)` in production code. Brings parity with iOS code: https://github.com/adobe/aepsdk-edgeidentity-ios/blob/main/Sources/Identity.swift#L123.